src/components/Game/Backspace/Backspace.scss

```diff
@@ -0,0 +1,41 @@
+:global {
+    @import "@fortawesome/fontawesome-free/scss/fontawesome";
+    @import "@fortawesome/fontawesome-free/scss/solid";
+}
+
+@import "../../../styles/abstracts/breakpoints";
+@import "../../../styles/abstracts/variables";
+
+.Backspace {
+    justify-self: self-end;
+}
+
+.Button {
+    width: $backspace-width;
+    height: $backspace-height;
+    border: none;
+    background: transparent;
+    cursor: pointer;
```

Comment:
by default you mean 'in our project' or 'in general'? in our project - yes, I didn't notice, it's in normalize.css. in general - it doesn't. I added the styling because I'm used to doing this.

Reply:
thank you normalize.css for saving my ass, of course I meant our project defaults :P

Reply:
😄